src/store/imagestore/types.ts

```diff
@@ -0,0 +1,14 @@
+export interface Image {
+    imageId: number,
```

Comment:
Please use autoformatter. (Default should be alt + shift + f keys.) Noticing some indentation is uneven.

Reply:
More comments if needed!